packages/google_maps_flutter/example/ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please don't commit your workspace settings.

Reply:
Fixed it by adding it to `.gitignore`, `git rm -r --cached . ` and `git add .`